namespace Exceptions
{
    internal class Program
    {

        static void Main(string[] args)
        {
            /* try
             {
                 using (StreamReader sr = File.OpenText("data.txt"))
                 {
                     Console.WriteLine($"The first line of this file is {sr.ReadLine()}");
                 }
             }
             catch (FileNotFoundException e)
             {
                 Console.WriteLine($"The file was not found: '{e.Message}'");
             }
             catch (DirectoryNotFoundException e)
             {
                 Console.WriteLine($"The directory was not found: '{e.Message}'");
             }
             catch (IOException e)
             {
                 Console.WriteLine($"The file could not be opened: '{e.Message}'");
             }catch (Exception e)
             {
                 Console.WriteLine(e.Message);
             }*/
            Console.WriteLine("Hello");
        }
    }
}